### ISSUE TITLE
[REQUEST] Gal*Gun: Double Peace

### ISSUE THREAD

**Issue by Thekenbaum** [Electry/VitaGrafix#84]
The game runs like utter garbage; the menus can get 60 fps but the game itself slows down to the teens.
Overclocking it puts it at around 30fps but slows down to 20-25 fps when the game has to render a lot.
The way this game handles lag is awful too, It's like they wanted to go the dk64 route of fixing lag but they forget to implement the part of the code that speeds up animations so your reticle zooms by really fast (making it hard to aim) while the girls slow down like their in an action scene.

TL:DR is it possible to lower the render resolution or lower the frame cap from 60 to 30 for the "lag fix" in the game doesn't act up nearly as badly?

**Comment by yeqing1990** [comment]
The game itself is not optimized, we hardly have much improvement. And the function of PSV itself is here.
